### PR TITLE
Update To Firebase 3.0

### PR DESCRIPTION
**Pull request by ljorgens**
Update to Firebase with authentication methods

### CODE REVIEW

**Comment by colinta** [comment]
Wow thank you for doing all this!  I'll go over the code real quick, but I'll merge it and try to get a new gem out soon.

I'm not working in RubyMotion these days, would you be interested in taking over this gem?

File: 3.4.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- [ ] Test login (facebook, github, twitter)
+- [ ] Fix demo app
+
+
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+    gem 'motion-firebase', github: 'soyoh/motion-firebase', branch: 'google-sdk'
```
Comment (colinta) [inline 3.4.md]: Are these instructions still relevant after this is merged?

**Comment by colinta** [comment]
@soyoh Is this based on your fork?  Would you like to take over running this gem?

File: 3.4.md (hunk 1 above)
Comment (ljorgens) [reply 3.4.md:13]: No that line should be updated to look at your repo

**Comment by ljorgens** [comment]
I found a small bug with create_user_and_login method.  Change line 83 from: 

 block.call(error, nil)

to:

 block.call(nil, error)

That should properly return the error

**Pull request merged**